src/AzureDBExperiences.ts

```diff
@@ -31,20 +31,25 @@ export function getExperienceFromApi(api: API): Experience {
     return info;
 }
 
-export function getExperienceLabel(resource: DatabaseAccount & Server): string {
-    const experience: Experience | undefined = tryGetExperience(resource);
-    if (experience) {
-        return experience.shortName;
+export function getExperienceLabel(databaseAccount?: DatabaseAccountGetResults, server?: Server): string | undefined {
```

Comment:
I think you can get rid of `server`. I only see you call `getExperienceLabel` once, and you're passing `undefined` for server anyways